CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 **BREAKING CHANGES:**
 
 * Removed `Summary.suggestions`.
+* Removed `bulk.dart` and `mini_sum.dart`.
```

Comment:
things in `lib/src` don't need to be documented.

Maybe mention `bin/bulk.dart`